packages/docs/src/tokens/Spacing/Spacing.padding.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+
+import './styles.scss';
+
+interface ISpacingPaddingComponent {
+  spacing: any;
+}
+
+function buildLists(spacing: any) {
```

Comment:
You should import the ISpacing interface from the styles package here instead of using any